jms/src/test/scala/docs/scaladsl/JmsConnectorsSpec.scala

```diff
@@ -1269,4 +1269,148 @@ class JmsConnectorsSpec extends JmsSpec {
 
     connection.close()
   }
+
+  "support request/reply with temporary queues and manual reply consumer" in withConnectionFactory() { connectionFactory =>
```

Comment:
This test shows my first attempt to combine a manually created `javax.jms.MessageConsumer` that consumes the replies send by the remote service.

Reply:
This example works and doesn't require us to support a custom consumer passed to `JmsConsumerSettings`, but is not very Akka Streams idiomatic.